src/sentry/seer/endpoints/project_autofix_automation_settings.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from typing import Any
+
+import sentry_sdk
+from django.db import router, transaction
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import ProjectPermission
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models.options.project_option import ProjectOption
+from sentry.models.project import Project
+from sentry.seer.autofix.constants import AutofixAutomationTuningSettings
+from sentry.seer.autofix.utils import (
+    AutofixStoppingPoint,
+    bulk_set_project_preferences,
+    get_project_seer_preferences,
+)
+from sentry.seer.models import SeerApiResponseValidationError
+
+
+class SeerAutofixSettingPutSerializer(serializers.Serializer):
```

Comment:
is it possible to combine some of this serializer logic with the organization endpoint? could we move them into a utils module? it looks to be the same at a quick glance